ui/shared/components/panel/family.jsx

```diff
@@ -46,6 +50,53 @@ const mapSampleDispatchToProps = (state, ownProps) => ({
 
 const FirstSample = connect(mapSampleDispatchToProps)(BaseFirstSample)
 
+const AnalystEmailDropdown = ({ load, loading, usersByUsername, onChange, value, ...props }) =>
+  <DataLoader load={load} loading={false} content>
+    <Select
+      loading={loading}
+      additionLabel="Assigned Analyst: "
```

Comment:
this prop shouldn't be used anymore, so regular selects don't have the addition option

Reply:
this is still true